src/clj/game/cards/basic.clj

```diff
@@ -0,0 +1,73 @@
+                                       (trigger-event state side :corp-click-draw (->> @state side :deck (take 1)))
+                                       (draw state side)
+                                       (swap! state update-in [:stats side :click :draw] (fnil inc 0))
+                                       (play-sfx state side "click-card")))}
+               {:label "Install 1 agenda, asset, upgrade, or piece of ice from HQ"
+                :async true
+                :effect (req (let [target-card (first targets)
+                                   server (second targets)]
+                               (corp-install state side (make-eid state {:source server :source-type :corp-install})
+                                             target-card server {:base-cost [:click 1] :action :corp-click-install})))}
+               {:label "Play 1 operation"
+                :async true
+                :effect (req (let [target-card (first targets)]
+                               (play-instant state side (make-eid state {:source :action :source-type :play})
+                                             target-card {:base-cost [:click 1]})))}
+               ]})
+
+(define-card "Runner Basic Action Card"
+  {:abilities [{:label "Gain 1 [Credits]"
+                :cost [:click]
+                :msg "gain 1 [Credits]"
+                :effect (req (gain-credits state side 1 :runner-click-credit)
+                             (swap! state update-in [:stats side :click :credit] (fnil inc 0))
+                             (trigger-event state side :runner-click-credit)
+                             (play-sfx state side "click-credit"))}
+               {:label "Draw 1 card"
+                :cost [:click]
+                :msg "draw 1 card"
+                :effect (req (wait-for (trigger-event-simult state side (make-eid state eid) :pre-runner-click-draw nil nil)
+                                       (trigger-event state side :runner-click-draw (->> @state side :deck (take 1)))
+                                       (draw state side)
+                                       (swap! state update-in [:stats side :click :draw] (fnil inc 0))
+                                       (play-sfx state side "click-card")))}
+               {:label "Install 1 program, resource, or piece of hardware from the grip"
+                :async true
+                :effect (req (let [target-card (first targets)]
+                               (runner-install state side (make-eid state {:source :action :source-type :runner-install})
+                                               target-card {:base-cost [:click 1]})))}
+               {:label "Play 1 event"
+                :async true
+                :effect (req (let [target-card (first targets)]
+                               (play-instant state side (make-eid state {:source :action :source-type :play})
+                                             target-card {:base-cost [:click 1]})))}
```

Comment:
You know, I don't think we need this anymore seeing as we now have these abilities. Better to make it just like other abilities and remove the whole `base-cost` hack.

Reply:
I left this in to display the correct messages. In the long run I'd like to remove all `:base-cost` arguments.